packages/modules/src/modules/banners/momentTemplate/settings.ts

```diff
@@ -39,7 +39,7 @@ export interface TickerStylingSettings {
 }
 
 export interface HeaderSettings {
-    textColour: string;
+    textColour?: string;
```

Comment:
There is a fallback `textColour` if one isn't provided in the settings, so this change alone is an improvement!